traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/models/ParkingItemDetails.java

```diff
@@ -76,9 +66,6 @@ private ParkingItemDetails(
         this.priceToPayOnline = priceToPayOnline;
         this.priceToPayOnsite = priceToPayOnsite;
         this.productType = productType;
-        this.categories = categories;
-        this.isWishlisted = isWishlisted;
-        this.providerTranslationAttribution = providerTranslationAttribution;
```

Comment:
Wee need the providerTranslationAttribution there for legal purposes. If translating through google tranlsate backend will populate this field and send it to us and we're obligated by contract with viator to display that attribution.